solvis/geometry.py

```diff
@@ -85,14 +118,37 @@ def create_surface(
 
 def bearing(point_a: Point, point_b: Point) -> float:
     """
-    Computes the bearing in degrees from the point A(a1,a2) to
-    the point B(b1,b2). Point(lat, lon))
-
-    Note that A and B are given in decimal.degrees
-
-    :param point_a: the first point
-    :param point_b: the second point
+    Computes the bearing in degrees from one Point to another Point.
+
+    Points are treated as `(lat, lon)` pairs of decimal degrees.
+
+    Parameters:
+        point_a: the first point
+        point_b: the second point
+
+    Returns:
+        A bearing in degrees
+
+    Raises:
+        ValueError: If the points are identical, they cannot be compared.
+
+    Examples:
+        ```py
+        >>> from shapely.geometry import Point
+        >>> from solvis.geometry import bearing
+        >>> loc_chc = Point(-43.53, 172.63)
+        >>> loc_akl = Point(-41.3, 174.78)
+        >>> bearing(loc_chc, loc_akl)
+        36.17316361836124
+        >>> bearing(loc_akl, loc_chc)
+        214.72263050205407
+        >>> bearing(loc_akl, loc_akl)
```

Comment:
@DrCuriosity this example is a bit long, perhaps omit the error scenario ?

Reply:
Fair :-) Error scenario omitted. We have a "Raises:" note to signpost that now, anyway.